components/CommentsWithData.js

```diff
@@ -107,6 +114,24 @@ class CommentsWithData extends React.Component {
       );
     }
 
+    if (!expenseComments?.comments) {
+      return loading || this.props.loadingLoggedInUser ? (
+        <LoadingPlaceholder height={76} borderRadius={8} />
+      ) : (
+        <MessageBox type="info" px={4}>
+          <Flex alignItems="center">
+            <PrivateInfoIcon size={42} withoutTooltip />
+            <P ml={3} fontSize="Paragraph" lineHeight="Paragraph">
+              <FormattedMessage
+                id="expense.privateCommentsWarning"
+                defaultMessage="The comments for this expense are private, you need to be authenticated as a host/collective admin or as an owner to see them."
```

Comment:
I guess `or as the expense owner` would be clearer.

Reply:
Indeed!

I got a similar suggestion from Alanna:
> Comments for this expense are private. You must be signed in as an admin or the expense submitter to view them.”